### PR TITLE
Added calificaciones view controller and support for round views

### PR DESCRIPTION
**Pull request by carooobregon**

### CODE REVIEW

File: CBMT/IntroCalificacionesViewController.swift [hunk 1]
```diff
@@ -24,6 +24,8 @@ class IntroCalificacionesViewController: UIViewController, UITableViewDataSource
         Alumno(nombre: "Lucia Cantu")
     ]
     
+    let cellColors = ["222B45","AC4040","FFC700", "11A05B", "FFB110"];
+    let colors = UIColor.systemRed
```
Comment (carooobregon) [inline CBMT/IntroCalificacionesViewController.swift:28]: need to delete this

File: CBMT/IntroCalificacionesViewController.swift [hunk 2]
```diff
@@ -24,6 +24,8 @@ class IntroCalificacionesViewController: UIViewController, UITableViewDataSource
         Alumno(nombre: "Lucia Cantu")
     ]
```
Comment (carooobregon) [inline CBMT/IntroCalificacionesViewController.swift:25]: btw- this array of Alumnos is a temporary solution until we get backend resolved

File: CBMT/IntroCalificacionesViewController.swift (hunk 1 above)
Comment (gabrielacorona) [reply CBMT/IntroCalificacionesViewController.swift:28]: 네, 알겠습니다

**Review by gabrielacorona — APPROVED** [review]
잘 봐요

**Pull request merged**